Sources/StreamVideo/WebRTC/v2/StateMachine/Stages/WebRTCCoordinator+Joined.swift

```diff
@@ -119,10 +119,6 @@ extension WebRTCCoordinator.StateMachine.Stage {
 
                     try Task.checkCancellation()
 
-                    await observeCallSettingsUpdates()
```

Comment:
why do we remove this?

Reply:
This method was here to forward CallSettings to the `publisher` PeerConnection. However, because it was relying on the Combine chain for updates, it was always a time-fracture behind.

Now, we are removing this and [moving](https://github.com/GetStream/stream-video-swift/blob/876bfe5b60a861f56dc54826a6415926fb9771d1/Sources/StreamVideo/WebRTC/v2/WebRTCStateAdapter.swift#L508) the `publiher` peerconnection update directly on the `WebRTCStateAdapter` next to when it updates its own CallSettings.